packages/block-editor/src/components/url-popover/index.js

```diff
@@ -16,10 +21,23 @@ function URLPopover( {
 	additionalControls,
 	children,
 	renderSettings,
-	position = 'bottom center',
+	placement = 'bottom',
 	focusOnMount = 'firstElement',
+	// Deprecated
+	position,
+	// Rest
 	...popoverProps
 } ) {
+	let computedPlacement = placement;
+	if ( position !== undefined ) {
+		deprecated( '`position` prop in wp.blockEditor.URLPopover', {
+			since: '6.2',
+			alternative: '`placement` prop',
+		} );
+
+		computedPlacement = positionToPlacement( position );
```

Comment:
Extreme nit, but with that order of logic, if someone passes both `position` and `placement`, the deprecated prop will take precedence. Should we do this only if `placement` is not specified?

Reply:
Since I moved the default value from `position` to `placement`, the `placement` prop can't really be `undefined` in the function's body. Therefore, in order to maintain backwards compat, I decided to implement the current logic.

I could refactor the logic and:

- move the default value assignment of `placement` from the prop destructuring to later in the function body
- make it so that when both `placement` and `position` are defined, `placement` takes the precedence

Let me know if this sounds like a good idea to you! If so, I'll go ahead and implement it

Reply:
What you suggested sounds good! I'll leave it to your judgment whether to implement it before merging.

Reply:
Pushed https://github.com/WordPress/gutenberg/pull/44391/commits/efd476f0d231c5ee17ee4ff9ba3a72df7b6cca15 with those changes